src/api.jl

```diff
@@ -115,13 +117,15 @@ end
 """
     complete_orbit(ds::DynamicalSystem, u0::AbstractArray{<:Real}, T::Real; kwargs...) → StateSpaceSet
 
-Complete the periodic orbit `po` of period `po.T`. For POs of discrete systems, it means iterating 
-the periodic point `po.T` times. For POs of continuous-time systems, it means integrating the system for 
-`po.T` time units with step `Δt`. For POs of discrete-time systems `Δt` must be equal to `1`. 
+Given point `u0` on the periodic orbit with period `T`, compute the remaining points of the 
+periodic orbit. For POs of discrete-time systems, it means iterating the periodic point 
+`po.T` times. For POs of continuous-time systems, it means integrating the system for 
+duration `po.T` with stepsize `Δt`.
 
 ## Keyword arguments
 
-* `Δt` : step size for continuous-time systems.
+* `Δt` : integration/iteration stepsize. For POs of discrete-time systems `Δt` must be 
+  equal to `1`.
```

Comment:
Do you mean continuous-time systems? The default values is `0.01` and user can change it. I like the idea that it should depend on the period. Isn't `T/50` or `T/100` better?

Reply:
Yes, let's go for T/100.

Reply:
notice that for this aspect you need to call `step!(ds, Δt, true)`. The third argument has to be `true` because adaptive solvers will typically not take steps less than 1/10th of a periodic orbit, at least to my experience.

Reply:
Function `complete_orbit` calls `trajectory` and not `step!` so I suppose that will be precise.